client/checkout/classic/upe-split.js

```diff
@@ -354,6 +355,21 @@ jQuery( function ( $ ) {
 				} else {
 					mountUPEElement( paymentMethodType, upeDOMElement );
 				}
+
+				if (
+					paymentMethodsConfig[ paymentMethodType ].countries.length
+				) {
+					maybeTogglePaymentMethod(
+						paymentMethodType,
+						paymentMethodsConfig[ paymentMethodType ].countries
+					);
+					$( '#billing_country' ).on( 'change', function () {
+						maybeTogglePaymentMethod(
+							paymentMethodType,
+							paymentMethodsConfig[ paymentMethodType ].countries
+						);
+					} );
+				}
```

Comment:
1. Should this block be moved to a separate function?

Reply:
2. Is it ok to add multiple listeners, or should map all PMs with restricted countries and handle them in a single listener?